app/client/src/widgets/InputWidgetV2/widget/index.tsx

```
@@ -641,7 +648,10 @@
   // TODO: Fix this the next time the file is edited
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   static getMetaPropertiesMap(): Record<string, any> {
-    return merge(super.getMetaPropertiesMap(), {
+    const baseMetaProperties = super.getMetaPropertiesMap();
```

Biome

[error] Using super in a static context can be confusing.

super refers to a parent class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)